appveyor.yml

```diff
@@ -0,0 +1,70 @@
+version: 1.0.{build}
+
+branches:
+  only:
+  - master
+  - /.*appveyor.*/
+  - /\d*\.\d*\.\d*/
+
+environment:
+  STACK_ROOT: C:\sr
+  STACK_VERSION: 1.9.3
+
+  matrix:
+    - STACK_YAML: stack.yaml
+    - STACK_YAML: stack-lts-6.yaml
+
+install:
+  - choco install -y haskell-stack --version %STACK_VERSION%
+  - stack setup > nul
+
+cache:
+  - C:\sr -> '%STACK_YAML%'
+  - .stack-work -> '%STACK_YAML%'
+  - dhall\.stack-work -> '%STACK_YAML%'
+  - dhall-json\.stack-work -> '%STACK_YAML%'
+  - dhall-text\.stack-work -> '%STACK_YAML%'
+  - dhall-bash\.stack-work -> '%STACK_YAML%'
+  
+build_script:
+  - stack build
+  - stack install --local-bin-path bin
+  - if /I "%APPVEYOR_REPO_TAG%" EQU "true" (set DEPLOY_TAG=%APPVEYOR_REPO_TAG_NAME%) else (set DEPLOY_TAG=%APPVEYOR_REPO_COMMIT:~0,5%)
+  - set DEPLOY_SUFFIX=%DEPLOY_TAG%-x86_64-windows.zip
+  - 7z a "bin\main\dhall-%DEPLOY_SUFFIX%" "%APPVEYOR_BUILD_FOLDER%\bin\dhall.exe"
+  # dhall-json cannot be built with lts-6
+  - if exist "%APPVEYOR_BUILD_FOLDER%\bin\dhall-to-json.exe" ( 7z a "bin\dhall-json-%DEPLOY_SUFFIX%" "%APPVEYOR_BUILD_FOLDER%\bin\dhall-to-json.exe" )
+  - if exist "%APPVEYOR_BUILD_FOLDER%\bin\dhall-to-yaml.exe" ( 7z a "bin\dhall-json-%DEPLOY_SUFFIX%" "%APPVEYOR_BUILD_FOLDER%\bin\dhall-to-yaml.exe" )
+  - 7z a "bin\dhall-text-%DEPLOY_SUFFIX%" "%APPVEYOR_BUILD_FOLDER%\bin\dhall-to-text.exe"
+  - 7z a "bin\dhall-bash-%DEPLOY_SUFFIX%" "%APPVEYOR_BUILD_FOLDER%\bin\dhall-to-bash.exe"
+
+for:
+-
+  matrix:
+    except:
+      - STACK_YAML: stack-lts-6.yaml
+
+  test_script:
+    - stack test dhall:tasty
+    - stack test dhall-json
+    - stack test dhall-text
+    - stack test dhall-bash
+
+artifacts:
+  - path: bin\main\dhall-*.zip
```

Comment:
Does this need to be inside of a `main` directory?  It seems like it would be more consistent to put it directly underneath `bin\` like the other projects

Reply:
Mmm, it is only a quick workaround to distinguish in file globs dhall itself from dhall-other-ones
But the zips generated for the deploy have the .exe at top level, without any dir. So the dir layout does not survive the build.

Reply:
Why not do an exact match (i.e. `"bin\main\dhall-%DEPLOY_SUFFIX%"`) instead of a wildcard match?

Reply:
Oh yeah that would be the right one, let me try just in case appveyor doesn't allow env vars in that position

Reply:
ci is happy, thanks for the suggestion!